test/module/shared_model/validators/block_validator_test.cpp

```diff
@@ -103,8 +107,61 @@ TEST_F(BlockValidatorTest, EmptyBlock) {
 TEST_F(BlockValidatorTest, InvalidBlock) {
   std::vector<shared_model::proto::Transaction> txs;
   txs.push_back(generateTx(false));
-  auto invalid_block = generateBlock(txs);
+  auto invalid_block =
+      generateBlock(txs, std::vector<shared_model::crypto::Hash>{});
 
   auto validation_result = validator_.validate(invalid_block);
   ASSERT_TRUE(validation_result.hasErrors());
 }
+
+/**
+ * @given block validator @and invalid block
```

Comment:
Copy-pasted description, please fix it and make more specific.

Reply:
Fixed